java/bogdana_maksiuta/src/com/zoolatech/lecture4/tasks/_3/StringModifier.java

```diff
@@ -0,0 +1,31 @@
+package com.zoolatech.lecture4.tasks._3;
+
+/**
+ * Create a functional interface StringModifier with a method modify that accepts a string
+ * and returns a modified string.
+ * In the main class create a method that accepts a string value
+ * and a lambda expression, applies the lambda to the value and outputs the result.
+ * In the main method create two lambda expressions that implement the interface:
+ * one should convert all characters to the lower case, second - to the upper case.
+ * Both lambdas should prepend a prefix “modified: ” before the modified values
+ * (e.g., “test” -> “modified: TEST”). Call the defined method with some string value and both lambdas.
+ * Call the same method with two method references that just convert all characters to the lower and upper case.
+
+ */
+@FunctionalInterface
+public interface StringModifier {
+    String modify(String value);
+}
+
+class Main {
+    static String value = "TeSt";
+    static void method (StringModifier stringModifier) {
+        System.out.println(stringModifier.modify(value));
+    }
+
+    public static void main(String[] args) {
+        System.out.println(value);
+        method((String str) -> "modified: " + str.toLowerCase());
```

Comment:
you don't need to define argument type and parentheses if a lambda expression contains only a single argument